### PR TITLE
Add image link of ROCm version.

### PR DESCRIPTION
**Pull request by liangadam**

### CODE REVIEW

File: doc/install/easy-install.md [hunk 1]
```diff
@@ -53,3 +53,8 @@ To pull the GPU version:
 ```bash
 docker pull ghcr.io/deepmodeling/deepmd-kit:2.0.0_cuda10.1_gpu
 ```
+
+To pull the ROCm version:
+```bash
+docker pull deepmodeling/dpmdkit-rocm:dp2.0.4-rocm4.5.2-tf2.6-lmp29Sep2021
```
Comment (njzjz) [inline doc/install/easy-install.md:59]: Why is it dp2.0.4?

Comment (liangadam) [reply doc/install/easy-install.md:59]: image

I‘m confused, too...

Comment (njzjz) [reply doc/install/easy-install.md:59]: What's the source code you built from? setuptools-scm will bump a patch version if you do some change to a stable version, but you can use an environment variable to assign the version.

**Comment by njzjz** [comment]
Also, the CI in the master branch is broken, so I changed the base branch to devel.

File: doc/install/easy-install.md (hunk 1 above)
Comment (liangadam) [reply doc/install/easy-install.md:59]: It explains the matter~I have made some adjustments because the gelu gradient definition conflicts with TF 2.6.

Comment (liangadam) [reply doc/install/easy-install.md:59]: Also cmakefiles need to be modified on ROCm 3.5 or higher.

**Review by wanghan-iapcm — APPROVED** [review]

**Pull request merged**